p2p/raintree/types/proto/raintree.proto

```diff
@@ -4,7 +4,7 @@ package raintree;
 option go_package = "github.com/pokt-network/pocket/p2p/types";
 
 message RainTreeMessage {
-  uint32 level = 1;
+  int32 level = 1;
```

Comment:
<img width="898" alt="Screenshot 2022-12-01 at 7 35 55 PM" src="https://user-images.githubusercontent.com/1892194/205209265-e11319d1-4bf2-4728-8090-e1aa87af513f.png">

Given my comment above, we should be able to revert this.